src/content/doc-surrealql/statements/upsert.mdx

```diff
@@ -2,17 +2,20 @@
 sidebar_position: 25
 sidebar_label: UPSERT
 title: UPSERT statement | SurrealQL
-description: The UPSERT statement can be used to modify records in the database if they already exist.
+description: The UPSERT statement can be used to insert records or modify records that already exist
 ---
 import Since from '@components/shared/Since.astro'
+import Tabs from "@components/Tabs/Tabs.astro";
+import TabItem from "@components/Tabs/TabItem.astro";
 
 # `UPSERT` statement
 
 <Since v="v2.0.0" />
 
-The `UPSERT` statement can be used to modify records in the database if they already exist. If a specified record ID does not exist, it will be created.
+The `UPSERT` statement can be used to insert records into the database, or to update them if they exist.
 
-This is different from the [`UPDATE`](/docs/surrealql/statements/update) statement, which will do nothing if a specified record ID does not exist.
+> [!NOTE]
+> In versions of SurrealDB between 2.0.0 and 2.0.5, an UPSERT statement was treated as an "UPDATE, otherwise INSERT" statement. It has since been changed to a statement which defaults to insertion, and updates otherwise. Please see the examples below for details.
```

Comment:
```suggestion
> In versions of SurrealDB between 2.0.0 and 2.0.4, an UPSERT statement was treated as an "UPDATE, otherwise INSERT" statement. It has since been changed to a statement which defaults to insertion, and updates otherwise. Please see the examples below for details.
```